build_conda_arm64.sh

```diff
@@ -4,7 +4,7 @@ cd $PYSAMDIR || exit
 source $(conda info --base)/etc/profile.d/conda.sh
 
 # get `CONDA_TOKEN` env variable, required for authentication
-source ~/.env || exit
+#source ~/.env || exit
```

Comment:
we could remove the `|| exit` part?

Reply:
Updated in [67aa4b5](https://github.com/NREL/pysam/pull/146/commits/67aa4b57c75bd549f0b6ce8c1a498c013456ce8e)